tico/experimental/quantization/passes/remove_weight_dequant_op.py

```diff
@@ -30,11 +30,16 @@
 )
 
 
-def get_data_from_buffers(exported_program: ExportedProgram, node: torch.fx.Node):
+def get_constant(exported_program: ExportedProgram, node: torch.fx.Node):
     assert isinstance(node, torch.fx.Node)
-    named_buffers = dict(exported_program.named_buffers())
-    buffer_name = exported_program.graph_signature.inputs_to_buffers[node.name]
-    return named_buffers[buffer_name]
+    if node.name in exported_program.constants:
+        return exported_program.constants[node.name]
+    elif node.name in exported_program.graph_signature.inputs_to_buffers:
+        buffer_name = exported_program.graph_signature.inputs_to_buffers[node.name]
+        named_buffer = dict(exported_program.named_buffers())
+        return named_buffer[buffer_name]
+    else:
+        raise RuntimeError("NYI constant")
```

Comment:
```suggestion
        raise RuntimeError(f"Cannot find following constant node: {node.name}")
```

(optional)
How about this?